stix/utils/nsparser.py

```diff
@@ -24,15 +24,13 @@ def update(self, ns_info):
         self.input_schemalocs.update(ns_info.input_schemalocs)
 
     def finalize(self, ns_dict=None, schemaloc_dict=None):
-        if not ns_dict:
-            ns_dict = {}
-
-        if not schemaloc_dict:
-            schemaloc_dict = {}
+        ns_dict = dict(ns_dict.iteritems()) if ns_dict else {}
+        schemaloc_dict = dict(schemaloc_dict.iteritems()) if schemaloc_dict else {}
```

Comment:
I personally find the previous style more intuitive, even though it's a couple more lines. I'm ok with this, though.